lib/vnet/vnet.go

```diff
@@ -154,8 +154,8 @@ type TUNDevice interface {
 	Close() error
 }
 
-// Manager holds configuration and state for the VNet.
-type Manager struct {
+// NetworkStack holds configuration and state for the VNet.
+type NetworkStack struct {
```

Comment:
there's a bunch of things we probably don't need to export anymore, but don't worry about that in this PR, we should probably just clean that up at some point